app/utils/theme.js

```diff
@@ -42,3 +42,15 @@ export function isThemeSwitchingEnabled(state) {
     const {config} = state.entities.general;
     return config.EnableThemeSelection === 'true';
 }
+
+// getHighlightStyleFromTheme
+
+import * as HighlightStyles from 'react-syntax-highlighter/styles/hljs';
```

Comment:
Just some style nitpicking, but can you move this up to be with the rest of the import statements?

Reply:
Done 👍